recipes/pysam/meta.yaml

```diff
@@ -23,17 +23,16 @@ requirements:
     - xz
     - bzip2
     - libdeflate
+    - libcurl
+    - openssl  # [not osx]
   run:
     - python
     - zlib
     - xz
     - bzip2
     - libdeflate
-    - curl
-
-test:
-  imports:
-    - pysam
+    - libcurl
+    - openssl  # [not osx]
 
```

Comment:
```suggestion
    command:
        python run_test.py
```

Reply:
Not sure whether one can actually include a test like this in such a PR @bioconda/core might know about this.